tests/test_workspace.py

```diff
@@ -35,12 +35,29 @@
 class WorkspaceGen2TestSuite(lsst.utils.tests.TestCase):
 
     def setUp(self):
-        self._testWorkspace = tempfile.mkdtemp()
+        self._testWorkspace = os.path.realpath(tempfile.mkdtemp())  # Avoid link problems
```

Comment:
Could you expand the comment to say "Use os.path.realpath to avoid link problems"?